code/__HELPERS/mobs.dm

```diff
@@ -498,6 +498,19 @@ GLOBAL_LIST_EMPTY(species_list)
 	if(!HAS_TRAIT(L, TRAIT_PASSTABLE))
 		L.pass_flags &= ~PASSTABLE
 
+/proc/passmob_on(target, source)
+	var/mob/living/L = target

```

Comment:
no need to restrict this proc to mobs, you could move these as well as the passtable helpers into atoms, and use them for the skateboard too instead of setting the flag manually